cloudaux/orchestration/aws/glacier.py

```diff
@@ -0,0 +1,52 @@
+from cloudaux.aws.glacier import describe_vault, get_vault_access_policy, list_tags_for_vault
+from cloudaux.decorators import modify_output
+from flagpole import FlagRegistry, Flags
+
+registry = FlagRegistry()
+FLAGS = Flags('DESCRIBE', 'POLICY', 'TAGS')
+
+
+@registry.register(flag=FLAGS.DESCRIBE)
+def _describe_vault(vault_name, **conn):
+    vault_data = describe_vault(vault_name=vault_name, **conn)
+    return {'VaultARN': vault_data['VaultARN'],
```

Comment:
You may want to create a : `get_base()` method with the flag of `FLAGS.BASE`. This would also contain a `_version` field, which should be included for all orchestration results.

Reply:
Done